docarray/array/storage/elastic/getsetdel.py

```diff
@@ -118,6 +118,29 @@ def _del_doc_by_id(self, _id: str):
             self._client.delete(index=self._config.index_name, id=_id)
         self._refresh(self._config.index_name)
 
+    def _del_docs_by_ids(self, _ids: Sequence[str]):
+        """Concrete implementation of base class' ``_del_doc_by_ids``
+
+        :param _ids: the ids of the documents to delete
+        """
+        body = []
+        for _id in _ids:
+            if self._doc_id_exists(id):
+                body.append({"delete": {"_index": self._config.index_name, "_id": _id}})
+        if len(body) > 0:
+            self._client.bulk(body)
+            self._refresh(self._config.index_name)
+
+    def _set_doc_value_pairs(
```

Comment:
I think instead of having basically empty implementations here these methods should not be implemented at all. That way the standard way provided by a super-class can kick in.

Or is the plan to implement all of these as part of this PR @dongxiang123 ?